WordPress/Classes/ViewRelated/Post/PostListViewController.swift

```diff
@@ -494,7 +494,23 @@ class PostListViewController: AbstractPostListViewController, UIViewControllerRe
     }
 
     func cell(_ cell: UITableViewCell, handleTrashPost post: AbstractPost) {
-        deletePost(post)
+        if (post.status == "trash") {
+            weak var weakSelf = self
+
+            let cancelText = NSLocalizedString("Cancel", comment: "Cancels an Action")
+            let deleteText = NSLocalizedString("OK", comment: "Deletes post permanently")
+            let messageText = NSLocalizedString("Delete Permanently", comment: "Deletes post permanently")
```

Comment:
Thanks for your comments! Really appreciate it. I think the other changes should be quite manageable, but how do I generate the Localizeable.strings for the other languages?